.gitlab/packaging/rpm.yml

```diff
@@ -45,6 +45,7 @@
 .package_rpm_agent_7:
   variables:
     RELEASE_VERSION: $RELEASE_VERSION_7
+    AGENT_MAJOR_VERSION: 7
```

Comment:
```suggestion
```

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/31004/commits/8a52d52c113056e5fad1f3d4bfc4968b66b25d2c